src/lib/checkpoint/cp_mgr.cpp

```diff
@@ -74,10 +74,17 @@ void CPManager::shutdown() {
     LOGINFO("Stopping cp timer");
     iomanager.cancel_timer(m_cp_timer_hdl, true);
     m_cp_timer_hdl = iomgr::null_timer_handle;
+    m_shutdown_initiated = true;
```

Comment:
Shouldn't the shutdown initiated be done in Homestore::shutdown()?

Reply:
Actually its for the cp_mgr destructor to notify the cp_flush to not initiate back to back CP.

Reply:
Renamed the shutdown to cp_shutdown as its notify the cp flush that cp mgr is shutting down.